src/main/java/frc/robot/utils/ReefAngleCalcUtil.java

```diff
@@ -1,85 +1,92 @@
 package frc.robot.utils;
 
-import java.util.TreeMap;
-
 import edu.wpi.first.math.Pair;
 import edu.wpi.first.math.geometry.Translation2d;
 import frc.robot.Constants;
+import java.util.TreeMap;
 
-// assuming heading is correct (facing head on)
 public class ReefAngleCalcUtil {
 
-    public enum ScoreTagPosition {
-        ID_A, // ID 20 || ID 8
-        ID_B, // ID 21 || ID 7
-        ID_C, // ID 22 || ID 6
-        ID_D, // ID 17 || ID 11
-        ID_F, // ID 18 || ID 10
-        ID_G, // ID 19 || ID 9
-    }
+  public enum ScoreTagPosition {
+    ID_A, // ID 20 || ID 8
+    ID_B, // ID 21 || ID 7
+    ID_C, // ID 22 || ID 6
+    ID_D, // ID 17 || ID 11
+    ID_E, // ID 18 || ID 10
+    ID_F, // ID 19 || ID 9
```

Comment:
just realized, we should label the sides based on the diagram Josiah sent, right?